src/index.js

```diff
@@ -1,3 +1,3 @@
 export Grid from './components/Grid';
-export Row from './components/Row';
-export Col from './components/Col';
+export * from './components/Row';
+export * from './components/Col';
```

Comment:
yes, that's correct - I wrote a spec to test that everything is exported as it should be.  I was on the fence about doing as you suggest vs the glob.  I went with the glob because it'd make future changes easier to make, but explicit is also a good concern.  Made an update